CMakeLists.txt

```diff
@@ -12,7 +12,10 @@ add_executable(
     deps/noise/noise.c
     deps/sqlite/sqlite3.c
     deps/tinycthread/tinycthread.c)
-
+find_package(OpenGL REQUIRED)
```

Comment:
This is a clever way to add more graphical elements to the project rather than drawing with glfw.